core/coreapi/interface/options/unixfs.go

```diff
@@ -124,13 +126,26 @@ func (unixfsOpts) RawLeaves(enable bool) UnixfsAddOption {
 	}
 }
 
+// InlineBlocks tells the adder to inline small blocks into CIDs
+func (unixfsOpts) InlineBlocks(enable bool) UnixfsAddOption {
+	return func(settings *UnixfsAddSettings) error {
+		settings.InlineBlocks = enable
+		return nil
+	}
+}
+
 // InlineLimit sets the amount of bytes below which blocks will be encoded
-// directly into CID instead of being stored and addressed by it's hash
+// directly into CID instead of being stored and addressed by it's hash.
+// Specifying this option implicitly enables InlineBlocks
+// Default: 32 bytes
 //
-// Note that while there is no hard limit on the number of bytes here, it should
-// be kept at something reasonably low like 32b (default for 'ipfs add')
+// Note that while there is no hard limit on the number of bytes, it should
+// be kept at a reasonably low value, like 64 bytes if you intend to display
+// these hashes. Larger values like 256 bytes will work fine, but may affect
+// de-duplication of smaller blocks
 func (unixfsOpts) InlineLimit(limit int) UnixfsAddOption {
 	return func(settings *UnixfsAddSettings) error {
+		settings.InlineBlocks = true
```

Comment:
On the other hand, what are the cases somebody (other than in the command implementation) would set this option and then not want to inline blocks? It feels a bit like saying 'I want to inline blocks up to 123 bytes and I want to inline blocks'

Reply:
A user might want to change the defaults. That is, I might want to create a set of default options that change the *default* inline limit without actually enabling inlining (unless requested by the user).

Note: I don't actually care one way or the other about this.

Reply:
Well, yeah. The options case makes enough sense, I'll go with that

Reply:
Yeah, I was thinking along the same line as @Stebalien.  Also, since I assume the idea is to switch the `ipfs add` command to use the new API it best to keep the behavior constant.